script.js

```diff
@@ -17,3 +17,372 @@ function closemenu() {
+  popwindow.id = 'popup-container';
+  popwindow.classList.remove('close1');
+  popwindow.classList.add('popup-container');
+}
+function popupwindow1() {
+  popwindow1.id = 'popup-container';
+  popwindow1.classList.remove('close1');
+  popwindow1.classList.add('popup-container');
+}
+function popupwindow2() {
+  popwindow2.id = 'popup-container';
+  popwindow2.classList.remove('close1');
+  popwindow2.classList.add('popup-container');
+}
+function popupwindow3() {
+  popwindow3.id = 'popup-container';
+  popwindow3.classList.remove('close1');
+  popwindow3.classList.add('popup-container');
+}
+function popupwindow4() {
+  popwindow4.id = 'popup-container';
+  popwindow4.classList.remove('close1');
+  popwindow4.classList.add('popup-container');
+}
+function popupwindow5() {
+  popwindow5.id = 'popup-container';
+  popwindow5.classList.remove('close1');
+  popwindow5.classList.add('popup-container');
+}
+function popupwindow6() {
+  popwindow6.id = 'popup-container';
+  popwindow6.classList.remove('close1');
+  popwindow6.classList.add('popup-container');
+}
+
+function popupclose() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow.classList.remove('popup-container');
+  popwindow.classList.add('close1');
+}
+function popupclose1() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow1.classList.remove('popup-container');
+  popwindow1.classList.add('close1');
+}
+function popupclose2() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow2.classList.remove('popup-container');
+  popwindow2.classList.add('close1');
+}
+function popupclose3() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow3.classList.remove('popup-container');
+  popwindow3.classList.add('close1');
+}
+function popupclose4() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow4.classList.remove('popup-container');
+  popwindow4.classList.add('close1');
+}
+function popupclose5() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow5.classList.remove('popup-container');
+  popwindow5.classList.add('close1');
+}
+function popupclose6() {
+  const mainc = document.querySelector('#main-container');
+  mainc.classList.remove('close1');
+  mainc.classList.add('main-container');
+  popwindow6.classList.remove('popup-container');
+  popwindow6.classList.add('close1');
+}
+
+popupwindow();
+popupclose();
+popupwindow1();
+popupclose1();
+popupwindow2();
+popupclose2();
+popupwindow3();
```

Comment:
- [ ] [OPTIONAL] Kindly use a function to avoid code repetition
> eg 

```
const populatemodal = (clanname, index) => {
    const seventhProject = document.querySelector(clanname);
seventhProject.addEventListener('click', () => {
  document.getElementById('modal-title').innerHTML = projectInformation[index].name;
  document.getElementById('description').innerHTML = projectInformation[index].description;
  document.getElementById('featuredImage').innerHTML = projectInformation[index].featuredImage;
  document.getElementById('technology1').innerHTML = projectInformation[index].technologies.technology1;
  document.getElementById('technology2').innerHTML = projectInformation[index].technologies.technology2;
  document.getElementById('technology3').innerHTML = ### projectInformation[index].technologies.technology3;
});
}
```

Reply:
@aimemalaika, I understand the observation, but for now I don't know how to deploy it in my code. I also feel that there are many repeating lines. I am going to research about it and try to improve my code at the weekend with your good suggestion.